pulp_container/app/migrations/0038_add_manifest_metadata_fields.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.10 on 2024-02-29 16:04
+import warnings
+
+from django.db import migrations, models
+
+
+def print_warning_for_initializing_image_nature(apps, schema_editor):
+    warnings.warn(
+        "Manifests containing additional metadata, like image manifest annotations and "
+        "image configuration labels, need to be updated. Please, run 'pulpcore-manager "
+        "init-image-nature' to extract and initiliaze the metadata from related artifacts."
```

Comment:
Update this to a more friendlier manner: If you are interested in initializing annotations and labels, run this django command...